metatron-app/metatron-discovery
# #1818 add mssql connection extension

### Description
add mssql connection extension

**Related Issue** : #1818 

### How Has This Been Tested?
1. Create connection
 - see MS-SQL connection added.

2. Create workbench from MS-SQL connection
 - execute query.

3. Create DataSource from MS-SQL connection

### Types of changes



### Checklist:



### Additional Context
